builds/jenkins-build-pack-sls-app/Jenkinsfile

```diff
@@ -728,46 +751,41 @@ def createSubscriptionFilters(config, accountDetails, env_key, credsId) {
   def doc = deploymentDescriptor ? readYaml(text: deploymentDescriptor) : [:]
   def functionsElem = doc['functions']
   // changing variable names appropriately
-  def serviceDomain = "${configLoader.INSTANCE_PREFIX}-${config['domain']}-${config['service']}-${environment_logical_id}"
```

Comment:
Since we are not using `createSubscriptionFilters` function anymore, it's time to get rid of this code!

Reply:
Done